### PR TITLE
Added the Mask Detection demo for tf.keras

### PR DESCRIPTION
**Pull request by guy1992l**
Mask Detection is the first demo covering the new mlrun.frameworks package. It is focusing on the deep learning supported frameworks and its main purpose is demonstrating how easy it is to use mlrun on an existent code. 

Tensorflow.Keras or PyTorch (PyTorch-Lightning will be added next), mlrun can be applied seamlessly to the data scientist code and with it, he gains the following features: 
* Auto logging for both MLRun and Tensorboard.
* Distributed training with Horovod.
* Versioning of model and custom objects files.
* Hyperparameters tuning.
* Additional framework related features (mainly PyTorch - callbacks mechanism, generic and flexible training and evaluation methods).
* Serverless model servers.

Note: PyTorch's version will be added soon.

### CODE REVIEW

**Comment by aviaIguazio** [comment]
@guy1992l please open the PR on 0.6.x-dev
